spec/support/lscl_configs/nginx_json_logstash.conf

```diff
@@ -0,0 +1,42 @@
+input {
```

Comment:
These files are a direct import from the `elastic/examples` repo. I'd rather not maintain them, but view them as 'in the wild' configs.